generators/entity-client/templates/angular/src/main/webapp/app/entities/service/entity.service.ts.ejs

```diff
@@ -99,6 +100,23 @@ export class <%= entityAngularName %>Service {
     }
     <%_ } _%>
 
+    add<%= entityAngularName %>ToCollectionIfMissing(<%= entityInstance %>Collection: I<%= entityAngularName %>[], ...<%= entityInstancePlural %>ToCheck: (I<%= entityAngularName %> | null)[]): I<%= entityAngularName %>[] {
+        const <%= entityInstancePlural %>: I<%= entityAngularName %>[] = <%= entityInstancePlural %>ToCheck.filter(isPresent);
```

Comment:
Can we simplify `ToCheck` parameter and construct assignment without `isPresent`?